_posts/2023-07-05-vanilla-os-orchid-devlog.md

```diff
@@ -0,0 +1,122 @@
+
+#### Method 1 (manual)
+
+To try out Apx v2, follow these instructions for a user-side installation:
+
+- Create a working directory (highly recommended to isolate Apx v2 from your system).
+
+- Obtain the latest Distrobox release (version [1.5.0.2](https://github.com/89luca89/distrobox/archive/refs/tags/1.5.0.2.tar.gz) or higher) and unpack it into a "distrobox" folder within your newly created work directory.
+
+- Download the latest Apx build from our [GitHub repository](https://github.com/Vanilla-OS/apx/releases/download/continuous/apx.tar.gz) and unpack the `apx` binary into `~/.local/bin`, renaming it to `apx2` to avoid collisions with previous versions.
+- Make the binary executable using the command `chmod +x apx2` from the correct directory.
+
+- Create the configuration file at `~/.config/apx/apx.json`:
+
+```json
+{
+  "apxPath": "/home/YOUR_USERNAME/.local/share/apx/",
+  "distroboxpath": "/home/YOUR_USERNAME/your/workdir/distrobox-folder/distrobox",
+  "storageDriver": "btrfs"
+}
+```
+
+After following these steps, you can start using Apx v2 by typing `apx2`.
+
+> Please note this is alpha software, and there may be bugs. We kindly request you to report any issues to our [GitHub repository](https://github.com/Vanilla-OS/apx/issues).
+
+Since this is a local installation of Apx, there won't be any initial contents. You can proceed manually by creating a package manager, a stack, and finally, a subsystem:
+
+```shell
+apx2 pkgmanagers new
+apx2 stacks new
+apx2 subsystems new
+apx2 <SUBSYSTEM_NAME> --help
+```
+
+To simplify your initial experimentation with Apx v2, you can access some starter resources from our [vanilla-apx-configs repository](https://github.com/Vanilla-OS/vanilla-apx-configs/). Copy the contents into `~/.local/share/apx/` to populate Apx with initial resources like stacks and package managers. All major distributions are covered.
+
+#### Method 2 (automated)
+
+We have created a script that automates the above steps. Make sure you have `git` and `curl` installed.
+
+```bash
+#!/bin/bash
+
+WORK_DIR="YOUR_WORK_DIR"  # <- edit this path
+
+mkdir -p "$WORK_DIR"
+cd "$WORK_DIR"
+
+curl -LO https://github.com/89luca89/distrobox/archive/refs/tags/1.5.0.2.tar.gz
+tar -xzf 1.5.0.2.tar.gz
+
+curl -LO https://github.com/Vanilla-OS/apx/releases/download/continuous/apx.tar.gz
+tar -xzf apx.tar.gz
+mv apx "$HOME/.local/bin/apx2"
+chmod +x "$HOME/.local/bin/apx2"
+
+mkdir -p "$HOME/.config/apx"
+echo '{
+  "apxPath": "'"$HOME/.local/share/apx/"'",
+  "distroboxpath": "'"$HOME/$WORK_DIR/distrobox-folder/distrobox/"'",
+  "storageDriver": "btrfs"
+}' > "$HOME/.config/apx/apx.json"
+
+git clone https://github.com/Vanilla-OS/vanilla-apx-configs.git "$WORK_DIR/vanilla-apx-configs"
+mv "$WORK_DIR/vanilla-apx-configs/stacks" "$HOME/.local/share/apx/"
+mv "$WORK_DIR/vanilla-apx-configs/package-managers" "$HOME/.local/share/apx/"
+
+echo "Installation completed. You can now use Apx v2 by running 'apx2'."
+apx2 --version
+```
+
```

Comment:
```suggestion

Now, you can start working with the created subsystem. For more information on subsystem commands, type `apx2 <SUBSYSTEM_NAME> --help`
```
@pulkitkrishna00 How about this?

Reply:
That, along with my suggestion below, will be good.